File: public/phase-2-20160620022203/JAVA/inmates.java
// Name : Inmates
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// n : number of prisoners
		// t : crime threshold
		// c : selected inmates
		// severity :  severity of the crime committed by each inmate
		int index;
		Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int n = Integer.parseInt(elements[0]);
		int t = Integer.parseInt(elements[1]);
		int c = Integer.parseInt(elements[2]);
		int[] severity = new int[n];
		String[] severity_elements = (in.nextLine()).split(" ");
		for(index=0;index<n;index++)
			severity[index] = Integer.parseInt(severity_elements[index]);


		// write your code here
		// store your results in `result`

		// output
		// Dummy Data
		int result=2;

		System.out.println(result);

	}
}

File: public/phase-2-20160620022203/JAVASCRIPT/inmates.js
// Name : Inmates
// input:
// n : number of prisoners
// t : crime threshold
// c : selected inmates
// severity :  severity of the crime committed by each inmate
var index,result;
var elements = readline().split(" ");
var n = parseInt(elements[0], 10);
var t = parseInt(elements[1], 10);
var c = parseInt(elements[2], 10);
var severity = [];
var severity_elements = readline().split(' ');
for(index=0;index<n;index++)
	severity.push(parseInt(severity_elements[index], 10));


// write your code here
// store your results in `result`

// Dummy Data
result = 2;


// output
print(result);
